### PR TITLE
stock_orderpoint_generator: add fields trigger and route_id

### PR DESCRIPTION
**Pull request by aayartsev**

### CODE REVIEW

File: stock_orderpoint_generator/models/orderpoint_template.py [hunk 1]
```diff
@@ -80,6 +80,22 @@ class OrderpointTemplate(models.Model):
         "scheduled action for every product in this list.",
     )
     auto_last_generation = fields.Datetime(string="Last Automatic Generation")
+    trigger = fields.Selection(
+        selection=[
+            ("auto", "Auto"),
+            ("manual", "Manual"),
```
Comment (rousseldenis) [inline stock_orderpoint_generator/models/orderpoint_template.py:87]: @aayartsev Could you explain the aim of this ? As manual triggering for orderpoint creation... is manual creation

Comment (aayartsev) [reply stock_orderpoint_generator/models/orderpoint_template.py:87]: @rousseldenis Value of this field will be copy to record wich will be created from this template.

**Review by geomer198 — APPROVED** [review]
LGTM!

File: stock_orderpoint_generator/models/orderpoint_template.py (hunk 1 above)
Comment (rousseldenis) [reply stock_orderpoint_generator/models/orderpoint_template.py:87]: @aayartsev Could you add your changes in the USAGE.rst ? Thanks

Comment (dessanhemrayev) [reply stock_orderpoint_generator/models/orderpoint_template.py:87]: Hi @rousseldenis  )
I have added our changes to USAGE.rst
please check)

**Pull request closed without merging**